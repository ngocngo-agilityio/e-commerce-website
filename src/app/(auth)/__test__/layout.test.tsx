import { render } from '@testing-library/react';

// Components
import AuthLayout from '../layout';

describe('AuthLayout', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <AuthLayout>
        <div>Children component</div>
      </AuthLayout>,
    );

    expect(container).toMatchSnapshot();
  });
});
