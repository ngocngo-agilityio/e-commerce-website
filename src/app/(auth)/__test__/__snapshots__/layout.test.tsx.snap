// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthLayout should match snapshot 1`] = `
<div>
  <div
    class="css-1sall7q"
  >
    <h1
      class="chakra-heading css-jouhmg"
    >
      <a
        href="/"
      >
        NorthStar
      </a>
    </h1>
    <div
      class="css-32spr2"
    >
      <div>
        Children component
      </div>
    </div>
  </div>
</div>
`;
